src/components/Misc/PostButtonModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Dispatch, ReactElement, SetStateAction } from "react";
+import { Dialog, DialogTitle, DialogContent, Button } from "@mui/material/";
+
+const styles = {
+  dialogPosition: {
+    position: "absolute",
+    bottom: "500px",
+  },
+};
+
+type ModalProps = {
+  children: ReactElement<any, any>;
+  openModal: boolean;
+  setOpenModal: Dispatch<SetStateAction<boolean>>;
+};
+
+export default function PostButtonModal(props: ModalProps) {
```

Comment:
Not sure how everyone feels about this, but we could also destructure our props directly in the brackets:
```suggestion
export default function PostButtonModal({children, openModal, setOpenModal}: ModalProps) {
```

Comes down to preference but I think as long as we pick something and use it across the app

Reply:
I have applied this change. It is better to destructure. Cleaner to read through and saves a line of code.